pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewSchema.java

```diff
@@ -190,6 +190,9 @@ public static List<GenericRecord> setValueInRecord(RowList rows, ViewDefinition
         if (e.getPrimitive() != null) {
           if (ViewApplicator.ID_TYPE.equals(e.getColumnInfo().getInferredType())) {
             currentRecord.put(e.getColumnInfo().getName(), e.getString());
+            // log.warn("Found a id column named: ", e.getColumnInfo().getName());
```

Comment:
I think you should remove these commented out lines.

Reply:
Done.